lib/elixir/lib/kernel.ex

```diff
@@ -3557,7 +3556,7 @@ defmodule Kernel do
   If the `Foo.Bar` module is moved somewhere else, the references to `Bar` in
   the `Foo` module need to be updated to the fully-qualified name (`Foo.Bar`) or
   an alias has to be explicitly set in the `Foo` module with the help of
-  `Kernel.SpecialForms.alias/2`.
```

Comment:
I am not a fan of those changes because when we see `alias/2` it usually means it is in the same module. I would prefer to be explicit in our docs. :)

Reply:
Fair enough, reverted!

I just thought that based on https://github.com/elixir-lang/ex_doc/issues/679 this is the way to go. Or maybe for Kernel docs themselves we're really explicit, but everywhere else it's ok to link to Kernel/Kernel.SpecialForms? Or we just do it on case by case basis? ;-)

Reply:
In Elixir itself, I would prefer to always be explicit. Not sure about other projects thoughts we maintain - although I completely understand the expectation users may have that it should autolink.